src/api.js

```diff
@@ -2087,6 +2087,91 @@ api.declare({
   return res.reply(result);
 });
 
+/** Get a worker-type from a provisioner */
+api.declare({
+  method:     'get',
+  route:      '/provisioners/:provisionerId/worker-types/:workerType',
+  name:       'getWorkerType',
+  stability:  API.stability.experimental,
+  output:     'workertype-response.json#',
+  title:      'Get a worker-type',
+  description: [
+    'Get a worker-type from a provisioner.',
+  ].join('\n'),
+}, async function(req, res) {
+  const {provisionerId, workerType} = req.params;
+
+  const wType = await this.WorkerType.scan({
```

Comment:
@jonasfj is scan the right choice here?

Reply:
Good catch. Since we're only returning at most 1 worker-type, this should probably use `load`.